lib/block.mli

```diff
@@ -94,6 +94,9 @@ val set_variables: t -> (string * string) list
 val unset_variables: t -> string list
 (** [unset_variable t] is the list of environment variable to unset *)
 
+val required_libraries: t -> (Library.Set.t, string) Result.result
```

Comment:
This should not return a result. Other functions in this module either raise to let the user known or ignore errors.

Reply:
Ok, I'll let it throw.